### PR TITLE
Amendments made to get the unit tests running

### PR DESCRIPTION
**Pull request by hwilliams1**
Amendments made to get the unit tests running

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -11,26 +11,23 @@
...
-        <!-- Spring -->
         <commons.lang.version>2.6</commons.lang.version>
         <java.version>1.8</java.version>
         <maven-compiler-plugin.version>3.8.1</maven-compiler-plugin.version>
         <maven-surefire-plugin.version>2.22.2</maven-surefire-plugin.version>
         <org.mapstruct.version>1.3.0.Final</org.mapstruct.version>
         <spring-boot-dependencies.version>2.1.15.RELEASE</spring-boot-dependencies.version>
         <spring-boot-maven-plugin.version>2.1.15.RELEASE</spring-boot-maven-plugin.version>
-        <start-class>uk.gov.companieshouse.chd.order.api.ChdOrderApiApplication</start-class>
+        <start-class>uk.gov.companieshouse.chdorderconsumer.ChdOrderConsumerApplication</start-class>
```
Comment (nabeelamirch) [inline pom.xml:25]: Is this correct?

Comment (hwilliams1) [reply pom.xml:25]: nope

File: pom.xml [hunk 2]
```diff
@@ -22,7 +22,7 @@
         <org.mapstruct.version>1.3.0.Final</org.mapstruct.version>
         <spring-boot-dependencies.version>2.1.15.RELEASE</spring-boot-dependencies.version>
         <spring-boot-maven-plugin.version>2.1.15.RELEASE</spring-boot-maven-plugin.version>
-        <start-class>uk.gov.companieshouse.chdorderconsumer.ChdOrderConsumerApplication</start-class>
+        <start-class>uk.gov.companieshouse.chdorderconsumer.ChdOrderApiApplication</start-class>
```
Comment (nabeelamirch) [inline pom.xml:25]: should chdorderconsumer be chdorderapi? Also which branch are we supposed to merging into, master or main?

**Review by YiannisPapastergiou — DISMISSED** [review]

File: pom.xml (hunk 2 above)
Comment (hwilliams1) [reply pom.xml]: it should be chd.order.api

Comment (hwilliams1) [reply pom.xml]: And we need to merge to master for the pipeline to work

File: pom.xml [hunk 3]
```diff
@@ -25,12 +25,9 @@
         <start-class>uk.gov.companieshouse.chd.order.api.ChdOrderApiApplication</start-class>
```
Comment (nabeelamirch) [inline pom.xml:25]: The other services put the service name as one word, e.g. item-handler is itemhandler, chd-order-consumer is chdorderconsumer. But I will approve.

**Review by nabeelamirch — APPROVED** [review]

**Pull request merged**